src/Common/src/Internal/NativeFormat/NativeFormatWriter.cs

```diff
@@ -200,7 +200,7 @@ public void PatchByteAt(int offset, byte value)
 
         public void WriteString(string s)
         {
-            // The actual bytes are only necessary for the final version during the growing plase
+            // The actual bytes are only necessary for the final version during the growing phase
```

Comment:
If you would like to fix this, I would prefer to fix this in the dotnet/runtime repo (src\coreclr\src\tools\Common\Internal\NativeFormat). This file is shared with the shipping runtime and we keep that one as the source of thruth (local changes here need to be backported manually and that just adds overhead).